### PR TITLE
[Fix] Fix normalization in restoration_video_inference

### PR DESCRIPTION
**Pull request by ckkelvinchan**
## Motivation
Most of the video SR models in MMEditing take inputs with range [0, 1], but TDAN takes inputs with range [-0.5, 0.5]. Therefore, `Normalize` should be added to the pipeline in `restoration_video_inference.py`.

## Modification
`demo_pipeline` is added in the configuration files of each VSR model. The demo will look for `demo_pipeline`.

## Remark
An additional modification is that the output images are named according to `args.filename_tmpl` and `args.start_idx`.

### CODE REVIEW

**Comment by innerlee** [comment]
A somewhat meta question... Why not add a test pipeline in configs and directly load from there?

**Comment by ckkelvinchan** [comment]
> A somewhat meta question... Why not add a test pipeline in configs and directly load from there?

As the pipeline for sliding-window framework (e.g.  `EDVR`, `TDAN`) is not quite suitable for inferencing a sequence (they have multiple LQs but a single GT).

**Comment by innerlee** [comment]
But the test pipeline do not have to be the same as val pipeline

File: mmedit/apis/restoration_video_inference.py [hunk 1]
```diff
@@ -38,35 +38,8 @@ def restoration_video_inference(model, img_dir, window_size, start_idx,
...
-            test_pipeline.append(op)
-
-    test_pipeline.extend([
-        dict(type='FramesToTensor', keys=['lq']),
-        dict(type='Collect', keys=['lq'], meta_keys=['lq_path', 'key'])
-    ])
-
     # build the data pipeline
-    test_pipeline = Compose(test_pipeline)
+    test_pipeline = Compose(model.cfg.demo_pipeline)
```
Comment (innerlee) [inline mmedit/apis/restoration_video_inference.py:42]: This should fall back to test_pipeline if no demo_pipeline exist, and to val_pipeline if neither demo_pipeline nor test_pipeline exist.

**Review by innerlee — APPROVED** [review]

**Pull request merged**